src/sentry/integrations/github/issues.py

```diff
@@ -26,6 +27,16 @@ def get_issue_url(self, key: str) -> str:
         repo, issue_id = key.split("#")
         return f"https://{domain_name}/{repo}/issues/{issue_id}"
 
+    def get_feedback_issue_body(self, event: GroupEvent) -> str:
+        body = "|  |  |\n"
+        body += "| ------------- | --------------- |\n"
+        for evidence in sorted(
+            event.occurrence.evidence_display, key=attrgetter("important"), reverse=True
+        ):
+            body += f"| **{evidence.name}** | {evidence.value} |\n"
+
```

Comment:
I think if we can hoist out the `message` evidence to be outside the table, that would be good.

some pseudo code for that idea is:
```
def thing():
  messages = [item in list if item.name == 'message']
  others = [item in list if item.name != 'message']

  body = ''
  for message in messages:
    body += messages

  body += 'tableheader'
  body += table_divider
  for evidence in others:
    body += evidence.value
```